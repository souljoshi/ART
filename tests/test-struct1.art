/* Test basic struct declaration */

struct intpair {
    int x;
    int y;
}

struct charpair {
    char x, y;
}

int main ()
{
    struct intpair  a;
    struct charpair b;

    a.x = 77;
    a.y = 18;

    b.x = 'A';
    b.y = 'q';

    printi(a.x);
    printi(a.y);

    printc(b.x);
    printc(b.y);
}