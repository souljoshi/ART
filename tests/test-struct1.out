77
18
Aq